setup.py

```diff
@@ -1,10 +1,21 @@
 #!/usr/bin/env python
 from setuptools import find_packages, setup
+import sys
 import versioneer
 
 README = open('README.rst', 'r').read()
 
 
+install_requires = [
+    'future>=0.14.0; python_version<"3"',
+    'futures; python_version<"3.2"',
+]
+
+if sys.version_info[0] == 2:
+    install_requires.append('ujson<=2.0.3; platform_system!="Windows"')
+else:
+    install_requires.append('ujson>3.0.0')
```

Comment:
```suggestion
    install_requires.append('ujson>=3.0.0')
```

There are available wheels for all OSes since ujson 3.0.0, to the constraint is wrong.